service-loadbalancer/template.cfg

```diff
@@ -61,6 +61,9 @@ defaults
     # default default_backend. This allows custom default_backend in frontends
     default_backend default-backend
 
+    # default load balancing algorithm
```

Comment:
This files is getting more complex. Can we add a unittest that puts in some fake data, and checks the output? In theory, it would be better to have an integration test that actually starts up haproxy and makes sure it doesn't crash, but anything is fine for now. you can do it in a follow up pr, or if you're pressed for time, just file a bug and we'll get to it somethime post kubernetes 1.1